interview-questions.md

```diff
@@ -7,52 +7,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
Interview questions are an opportunity to share your knowledge, your experience, how you trouble shoot.